src/providers/secondary-media-loader.ts

```diff
@@ -0,0 +1,59 @@
+import ILoader = KalturaPlayerTypes.ILoader;
+
+const {RequestBuilder} = KalturaPlayer.providers;
+
+export class SecondaryMediaLoader implements ILoader {
+  _parentEntryId: string = '';
+  _requests: any[] = [];
+  _response: any = {};
+
+  static get id(): string {
+    return 'dualscreen';
+  }
+
+  /**
+   * @constructor
+   * @param {Object} params loader params
+   */
+  constructor(params: any) {
+    this._parentEntryId = params.parentEntryId;
+    const headers: Map<string, string> = new Map();
+    const request = new RequestBuilder(headers);
+    const INCLUDE_FIELDS = 1;
```

Comment:
Why did we choose to not use this as import from provider at the end?

Reply:
We thought that exporting the enums from providers should be done with thinking and in a more general manner.
So we decided for now to handle this locally as it minor

Reply:
ok, but still please address the nested accessor which I commented here https://github.com/kaltura/playkit-js-dual-screen/pull/6/files#r649123203 (we have utility in player for this - or use the .? and make sure babel transpiles it correctly)